modules/javafx.graphics/src/main/java/javafx/scene/image/Image.java

```diff
@@ -63,34 +65,40 @@
  * The {@code Image} class represents graphical images and is used for loading
  * images from a specified URL.
  *
- * <p>
- * Supported image formats are:
- * </p>
+ * <p>Supported image formats are:
```

Comment:
Can you revert this formatting change to avoid unnecessary diffs? (I mean the one that moved the sentence up to be on the same line as the `<p>`. The removal of the unneeded `</p>` is fine)

The reason for this is to avoid extra diffs in public API. It will make it easier to see what needs to go into the CSR.